core/Azure.Mcp.Core/tests/Azure.Mcp.Core.UnitTests/Areas/Server/Commands/Runtime/McpRuntimeTests.cs

```diff
@@ -258,10 +258,6 @@ public async Task CallToolHandler_DelegatesToToolLoader()
         mockTelemetry.Received(1).StartActivity(ActivityName.ToolExecuted, Arg.Any<Implementation?>());
         Assert.Equal(ActivityStatusCode.Ok, activity.Status);
 
```

Comment:
Were these lines failing the unit tests? I wanted to make sure we were including that tag in the telemetry